recipes/default.rb

```diff
@@ -16,3 +16,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+#
+# TODO
+# - Set up alfred user/group
+# - Add ssh key
+#      https://github.com/osuosl-cookbooks/osl-mirror/blob/master/recipes/default.rb#L73-L94
+#      ohai 'reload_passwd' fixes bug in ssh-keys cookbook
+# - Install Java
+#
+include_recipe 'java'
+
+group 'alfred' do
+  action :create
+end
+
+user 'alfred' do
+  supports manage_home: true
+  gid 'alfred'
+  system true
+  shell '/usr/bin/sh'
```

Comment:
This should be `/bin/bash` or `/bin/sh` if we choose to specify.
But I think taking the system default would be fine.